### ISSUE TITLE
System tray icon causes frames to be dropped during Steam games

### ISSUE THREAD

**Issue by childoftheabyss** [DeaDBeeF-Player/deadbeef#2983]
### Steps to reproduce the problem
1. Open deadbeef.
2. Disable "Hide system tray icon".
3. Open a game on Steam (e.g.: Rocket League).

### What's going on? Describe the problem in as much detail as possible.
Every time that I open deadbeef before or during a game session (Steam games, especially Rocket League and Sniper Elite using Proton) with the tray icon enabled the game continually stutters: small but perceptible frames skipping (the game framerate is not smooth anymore). The funny thing, the performance indicator for Rocket League at least doesn't show frames being dropped, with the minimum FPS very close to 144 (~141) whatever deadbeef or system tray icon on or off. However, my eyes sees the game stuttering affecting the game experience. The feeling is that some frames are being skipped to be shown on screen, but without affect the FPS indicator. I suspect some rendering issues with Gnome (I have more icons in system tray, and only happens specifically when I open deadbeef). Disabling the deadbeef system tray totally solves the problem. In Sniper Elite 3 the same effect occurs, but it doesn't have a performance overlay for me to check. I don't know how any of these works internally, but because the FPS indicator remains stable is probably after the GPU rendering. 

As a interesting note, I know the Gnome has a behavior when I receive some system notification during gaming that to display the notification balloon the game window quickly minimizes and it is restored, but at least the FPS indicator drops as well. 

### Information about the software:

Deadbeef version: 1.9.5
OS: PopOS 22.04 LTS
Gnome 42.9
Windowing system X11
GPU Nvidia 3060 Ti

**Comment by Oleksiy-Yakovenko** [comment]
I don't know for sure why this could be happening, but check whether your titlebar is displaying current track playback position. This is also reflected in the tray tooltip, so it may affect performance.

**Comment by Oleksiy-Yakovenko** [comment]
Btw, is the system tray icon of deadbeef visible? It was removed from GNOME a long time ago...

**Comment by childoftheabyss** [comment]
Yes, it was displaying the current playback position, but I removed from title bar and the frame drop continued. Also, I changed the UI plugin from GTK2 to GTK3 and the problem persisted.

About the icon, it is visible like this when enabled:

![image](https://github.com/DeaDBeeF-Player/deadbeef/assets/75948189/8a425471-5ef5-4a5a-8e67-e1a51554bf43)

But the tray tooltip has no information about time:
![image](https://github.com/DeaDBeeF-Player/deadbeef/assets/75948189/91740702-e1bd-4f5e-bee0-8c7aa3624fe1)
